package.json

```diff
@@ -18,12 +18,15 @@
     "gulp-babel": "^5.1.0",
     "gulp-flatten": "^0.1.1",
     "gulp-util": "^3.0.4",
-    "jest-cli": "^0.4.5",
+    "jest-cli": "^0.5.0",
     "merge-stream": "^0.1.8",
     "object-assign": "^3.0.0",
     "run-sequence": "^1.1.0",
     "through2": "^2.0.0"
   },
+  "engines": {
+    "node": ">=2"
```

Comment:
Actually 1 place that might use this info is TravisCI. Just confirm in relay what version gets run both before and with a PR to remove this line.